src/postcapture.rs

```diff
@@ -93,6 +93,17 @@ impl PostCaptureAction for CopyToClipboard {
         let clipboard = display.clipboard();
 
         clipboard.set_texture(&gdk::Texture::for_pixbuf(&pixbuf));
+
+        {
+            let settings = gio::Settings::new("kc.kcshot");
+            let test = settings.strv("test");
+
+            tracing::info!("Before");
+            for string in test {
+                tracing::debug!("{}", string);
+            }
+            tracing::info!("After");
+        }
```

Comment:
This gets removed in a later commit, and the test string array isn't useful, so why not just drop this commit? (eac1823ed70a59b70722d59afb8a4325b50f0668)